.build/run-python-dtests.sh

```diff
@@ -73,7 +72,7 @@ if [ "${java_version}" -eq 17 ] && [[ "${target}" == "dtest-upgrade" ]] ; then
     exit 1
 fi
 
-python_version=$(python -V | awk '{print $2}' | awk -F'.' '{print $1"."$2}')
+python_version=$(python -V 2>&1 | awk '{print $2}' | awk -F'.' '{print $1"."$2}')
```

Comment:
Since we removed the 3.6 version, should we add a constraint here that version >= 3.7?

Reply:
`cassandra_regx_supported_versions="^(2.2|3.0|3.11|4.0|4.1)(.[0-9]+)?$"`

Like this one in the accompanying script in the cassandra-builds :-)

Reply:
done in https://github.com/apache/cassandra/compare/90182972ebc0dee7f936058f222d2df966893d1d..557f90ed5ccdaa1e86459e135017ef96cb64bc4b